package.json

```diff
@@ -52,6 +53,10 @@
       "prettier --write"
     ]
```

Comment:
if we add this block of code here under our lint staged then husky will automatically run eslint on staged files only so we don't have to rely on folks doing it manually each time before they commit:

`    "src/**/*.+(js|json|ts|tsx)": [
      "eslint"
    ]
`

Can you update the readme with this as well similar to how you described the prettier/husky/npm run lint?

Reply:
Updated package.json to include automated eslinting during commit process. Updated readme to also highlight this process